Library/Homebrew/formula.rb

```diff
@@ -1073,13 +1073,13 @@ def self.core_files
   # an array of all tap {Formula} names
   # @private
   def self.tap_names
-    @tap_names ||= Tap.flat_map(&:formula_names).sort
+    @tap_names ||= Tap.reject(&:core_tap?).flat_map(&:formula_names).sort
```

Comment:
These methods are there to keep backward compatibility. I plan to deprecate `Fromula.tap_*` methods in the future.